### PR TITLE
Pal 131 move documentation to new repo 2nd draft

### PR DESCRIPTION
**Pull request by m78233**
Ready to run the gauntlet.  Changes are based on feedback from Jo.  Attached is a note with the feedback and comments.

### CODE REVIEW

**Comment by m78233** [comment]
Notes with feedback and comments

[2nd Draft Notes.pdf](https://github.com/gchq/Palisade/files/4586547/2nd.Draft.Notes.pdf)

File: README.md [hunk 1]
```diff
@@ -17,84 +17,43 @@ limitations under the License.
...
-Palisade is still in the early stages of development.
-At the moment there are a few simple examples to show how Palisade could work - it is not production ready.
-The description of Palisade below indicate the high level goals of the framework. 
-</span>
-
-## Documentation
-
-The documentation for the latest release can be found [here](https://gchq.github.io/Palisade).
 
+####Introduction
```
Comment (dev930018) [inline README.md:25]: Needs a space between #### and Introduction... see github render of this markdown

File: README.md [hunk 2]
```diff
@@ -17,84 +17,43 @@ limitations under the License.
...
-
-The examples may have additional prerequisites
-
-<span style="color:red">
-We do not currently support Windows as a build environment, If you are running on Windows then you will need this: Microsoft Visual C++ 2010 SP1 Redistributable Package
-</span>
-
 
-## Getting started
+###Additional Information
```
Comment (dev930018) [inline README.md:48]: Needs a space

File: README.md [hunk 3]
```diff
@@ -17,84 +17,43 @@ limitations under the License.
 
 
 # <img src="logos/logo.svg" width="180">
+###A Tool for Complex and Scalable Data Access Policy Enforcement
```
Comment (dev930018) [inline README.md:20]: Needs a space

File: README.md (hunk 2 above)
Comment (m78233) [reply README.md]: Thanks,  missed that

File: README.md (hunk 1 above)
Comment (m78233) [reply README.md]: Added the space.

File: README.md (hunk 2 above)
Comment (m78233) [reply README.md]: Added the space

File: README.md (hunk 3 above)
Comment (m78233) [reply README.md]: Added the space

File: README.md [hunk 4]
```diff
@@ -15,14 +15,13 @@ See the License for the specific language governing permissions and
...
-
 # <img src="logos/logo.svg" width="180">
-###A Tool for Complex and Scalable Data Access Policy Enforcement
+### A Tool for Complex and Scalable Data Access Policy Enforcement
 
 Palisade aims to provide a stable framework for defining and enforcing policies for accessing data.
 
 
-####Introduction
+#### Introduction
```
[the remark this answers is not in the document]
Comment (m78233) [reply README.md:24]: Put the bullet points in.

[the remark this answers is not in the document]
Comment (m78233) [reply README.md:24]: Replaced "this" with "any".  Now it reads "to be sanitised for any particular query"

[the remark this answers is not in the document]
Comment (m78233) [reply README.md:24]: Added the "to" .

[the remark this answers is not in the document]
Comment (m78233) [reply README.md:24]: Re-inserted changes to this paragraph.  Should be a better read.

[the remark this answers is not in the document]
Comment (m78233) [reply README.md:24]: Re-inserted changes to this paragraph.  Should be a better read.

File: README.md [hunk 5]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
 
-Policies are set per resource and are not changed by the user or context.  However, the policies can use the information about the user and context to decide how the data needs to be sanitised for this particular query.  This can be applied on a per field basis. This means that fields in a record can be redacted or masked based on: the contents of the record; the user accessing the record; and the contextual information given by the user or the system at the time of the query.
-
+Policies are set per resource and are not changed by the user or context.  However, the policies can use the information about the user and context to decide how the data needs to be sanitised for any particular query.  This can be applied on a per field basis. This means that fields in a record can be redacted or masked based on: 
+* the contents of the record
+* the user accessing the record 
+* the contextual information given by the user or the system at the time of the query.
```
Comment (ac74475) [inline README.md:34]:
```suggestion
* the contextual information provided by the user or the system at the time of the query.
```

File: README.md [hunk 6]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
 
 
-Palisade's aim is to provide a stable framework for defining and enforcing policy and auditing requirements for data access requests while at the same time, not limiting the solution any one platform ecosystem.  This will allow Palisade to be adapted into existing solutions that already do work in the different ecosystems such as Hadoop, Kubernetes, AWS, Azure, Google.io, etc.  Palisade does that by sharing with the community the task of creating connectors to new data storage technologies and the connectors from new data processing technologies into Palisade. 
 
+Palisade's aim is to provide a stable framework for defining and enforcing policy and auditing requirements for data access requests, while ,at the same time, not limiting the solution to any single platform ecosystem. Palisade can be adapted into existing solutions that already work in the different ecosystems such as Hadoop, Kubernetes, AWS, Azure and Google Cloud.  This is achieved by sharing with the community the tasks of creating connectors between Palisade and new technologies and solutions for storing data and for processing data.
```
Comment (ac74475) [inline README.md:38]:
```suggestion
Palisade's aim is to provide a stable framework for enforcing policy and auditing requirements for data access requests, while, at the same time, not limiting the solution to any single platform ecosystem. Palisade can be deployed into many different ecosystems such as Hadoop, Kubernetes, AWS, Azure and Google Cloud. As adoption of Palisade grows then burden of creating new connectors between Palisade and new technologies for storing and processing data can be shared across the community.
```

File: README.md [hunk 7]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
-###A Tool for Complex and Scalable Data Access Policy Enforcement
+### A Tool for Complex and Scalable Data Access Policy Enforcement
 
 Palisade aims to provide a stable framework for defining and enforcing policies for accessing data.
 
 
-####Introduction
-Modern organisations can have multiple datasets that are each managed and accessed according to different policies and procedures. This can make it complicated for analysis while still ensuring reglatory compliance.  Most technologies "out of the box" do not provide the necessary solution for managing these complex issues for data access. This leads to data being siloed and often accesses being restricted with the outcome that the insight from data can be lost.   
+#### Introduction
+Modern organisations can have multiple datasets that are each managed and accessed according to different policies and procedures. This can make it complicated for analysis while still ensuring regulatory compliance.  Most technologies "out of the box" do not provide the necessary solution for managing these complex issues for data access. This leads to access to data being overly restrictive with potential insights from data being lost.
```
Comment (dev930018) [inline README.md:25]: As these are markdown files, the general consensus is that it should stylistically be presented inside the file as one sentence per line.
See [this thread](https://github.com/prettier/prettier/issues/3302) discussing pros and cons, the biggest here being git diffs work per line.

File: README.md [hunk 8]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
+#### Introduction
+Modern organisations can have multiple datasets that are each managed and accessed according to different policies and procedures. This can make it complicated for analysis while still ensuring regulatory compliance.  Most technologies "out of the box" do not provide the necessary solution for managing these complex issues for data access. This leads to access to data being overly restrictive with potential insights from data being lost.   
 
 Palisade can provide a solution to this problem.  It consists of a set of services for managing and enforcing complex data access policies across different datasets and platforms.  These services are able to provide modular data access controllers and independent peripheral services to allow policies to be integral to the process a client uses to access data.
```
Comment (dev930018) [inline README.md:27]: Being super picky, this should be "Palisade can provide a solution to this problem; it consists..."

File: README.md [hunk 9]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
 
 Palisade can provide a solution to this problem.  It consists of a set of services for managing and enforcing complex data access policies across different datasets and platforms.  These services are able to provide modular data access controllers and independent peripheral services to allow policies to be integral to the process a client uses to access data.  
 
 Consider the example of a set of datasets may have been assembled for specific purposes.   For a given dataset, only particular types of query are permitted for certain types of user.  Also, within each dataset a specific user may only be permitted to access a particular subset of the records. Palisade allows the policies for accessing datasets, redacting records and executing the query to be specified independently of the physical data and the platform hosting the data.  This will enable the clients to perform queries across the datasets while still conforming to company and regulatory policies.
```
Comment (dev930018) [inline README.md:29]: First -> Second sentence is triple-spaced, but see comment on one-sentence-per-line

Comment (dev930018) [inline README.md:29]: Second sentence, "particular types of queries"

File: README.md [hunk 10]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
 
-Policies are set per resource and are not changed by the user or context.  However, the policies can use the information about the user and context to decide how the data needs to be sanitised for this particular query.  This can be applied on a per field basis. This means that fields in a record can be redacted or masked based on: the contents of the record; the user accessing the record; and the contextual information given by the user or the system at the time of the query.
-
+Policies are set per resource and are not changed by the user or context.  However, the policies can use the information about the user and context to decide how the data needs to be sanitised for any particular query.  This can be applied on a per field basis. This means that fields in a record can be redacted or masked based on:
```
Comment (dev930018) [inline README.md:31]: 3rd sentence, hyphenate "per-field"

File: README.md [hunk 11]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
+* the user accessing the record 
+* the contextual information provided by the user or the system at the time of the query.
 
 
-Palisade's aim is to provide a stable framework for defining and enforcing policy and auditing requirements for data access requests while at the same time, not limiting the solution any one platform ecosystem.  This will allow Palisade to be adapted into existing solutions that already do work in the different ecosystems such as Hadoop, Kubernetes, AWS, Azure, Google.io, etc.  Palisade does that by sharing with the community the task of creating connectors to new data storage technologies and the connectors from new data processing technologies into Palisade. 
 
+Palisade's aim is to provide a stable framework for enforcing policy and auditing requirements for data access requests, while, at the same time, not limiting the solution to any single platform ecosystem. Palisade can be deployed into many different ecosystems such as Hadoop, Kubernetes, AWS, Azure and Google Cloud. As adoption of Palisade grows then burden of creating new connectors between Palisade and new technologies for storing and processing data can be shared across the community.
```
Comment (dev930018) [inline README.md:39]: This weird diff is why one-per-line is preferred

File: README.md [hunk 12]
```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
...
```
Comment (ac74475) [inline README.md:38]:
```suggestion
Palisade's aim is to provide a stable framework for enforcing policy and auditing requirements for data access requests, while, at the same time, not limiting the solution to any single platform ecosystem. Palisade can be deployed into many different ecosystems such as Hadoop, Kubernetes, AWS, Azure and Google Cloud. As adoption of Palisade grows, then the burden of creating new connectors (between Palisade and new technologies for storing and processing data) can be shared across the community.
```

File: README.md [hunk 13]
```diff
@@ -43,12 +44,11 @@ Palisade's aim is to provide a stable framework for defining and enforcing polic
 If required, Palisade allows an organisation to use centralised services which each deployment of Palisade's data access services can share. This means that your data access policies, audit logs, user account details etc. don't need to be duplicated on every system. Of course, Palisade can still be deployed locally on a per platform basis.
```
Comment (dev930018) [inline README.md:44]: last sentence hyphenate "per-platform"

**Pull request merged**